test/e2e/tests/test_db_cluster_parameter_group.py

```diff
@@ -82,8 +83,14 @@ def test_create_delete_aurora_mysql5_7(self):
                 "value": "prod",
             }
         ]
+        new_params = {
+            "autocommit": "1",
+        }
         updates = {
-            "spec": {"tags": new_tags},
+            "spec": {
+                "tags": new_tags,
+                "parameterOverrides": new_params,
+            },
```

Comment:
Since this implementation is exceptionally complex - i'd suggest testing more than just one parameter.